### PR TITLE
Remove debug logging 

### PR DESCRIPTION
**Pull request by charliec443**
**What this PR does / why we need it**:
Fixes a typo
**Which issue(s) this PR fixes**:

**Does this PR introduce a user-facing change?**:

```release-note
None
```

### CODE REVIEW

File: sdk/python/feast/feature_view.py [hunk 1]
```diff
@@ -169,7 +169,7 @@ def to_proto(self) -> FeatureViewProto:
             ttl_duration = Duration()
             ttl_duration.FromTimedelta(self.ttl)
 
-        print(f"Stream soruce: {self.stream_source}, {type(self.stream_source)}")
+        print(f"Stream source: {self.stream_source}, {type(self.stream_source)}")
```
Comment (achals) [inline sdk/python/feast/feature_view.py:172]: I actually think we can delete this line altogether!

Comment (achals) [reply sdk/python/feast/feature_view.py:172]: (i think I left it in while running/debugging locally)

Comment (chappers) [reply sdk/python/feast/feature_view.py:172]: Haha - not a problem. This was when I was testing the offline config.

**Comment by achals** [comment]
looks good to me, you may have to address the [DCO check that's currently failing](https://github.com/feast-dev/feast/pull/1678/checks?check_run_id=2967251584).

**Review by woop — APPROVED** [review]

**Comment by achals** [comment]
@charliec443 you may need to rebase your changes, I think there was a pandas version fix that would solve the test failures

**Comment by charliec443** [comment]
@achals thanks for following up - I have rebased.

**Pull request merged**